go.mod

```diff
@@ -121,7 +123,8 @@ require (
 	github.com/hashicorp/hcl/v2 v2.16.2
 	github.com/hashicorp/hcp-link v0.1.0
 	github.com/hashicorp/hcp-scada-provider v0.2.1
-	github.com/hashicorp/hcp-sdk-go v0.23.0
+	github.com/hashicorp/hcp-sdk-go v0.70.0
+	github.com/hashicorp/hcp-vault-engine-poc v0.0.0-20231023141023-cbc8f2aa979f
```

Comment:
I get: go: github.com/hashicorp/hcp-vault-engine-poc@v0.0.0-20231018143610-d4661a2768ef: invalid version: unknown revision d4661a2768ef

Reply:
Oh right. The name has changed, but my vendor was outdated. Let me update it.